diff -ruN libtrash-1.1.old/src/Makefile libtrash-1.1/src/Makefile
--- libtrash-1.1.old/src/Makefile	Sun May 26 20:27:13 2002
+++ libtrash-1.1/src/Makefile	Sun Jun  2 12:02:41 2002
@@ -28,7 +28,7 @@
 install-libtrash: 
 	TRASH_OFF=YES install libtrash.so.${VERSION} ${INSTLIBDIR}/libtrash.so.${VERSION}
 	TRASH_OFF=YES install libtrash.conf.sys --mode a=r ${SYSCONFFILE}
-	ldconfig	
+	
 
 clean:
 	rm -f *~
